src/components/constants.ts

```diff
@@ -2,6 +2,7 @@ export const PLAYER_MINIMUM_GAMES_REQUIRED = 10;
 export const PLAYER_MINIMUM_WINS_REQUIRED = 5; // Used for average win turn
 export const COMMANDER_MINIMUM_GAMES_REQUIRED = 5;
 export const NEW_PLAYER_HIGHLIGHT_DAYS = 7; // Used for highlighting newly qualified players in the playerOverview list
+export const NUMBER_OF_PLAYERS_FOR_VALID_GAME = 4; // Used for filtering out non four player games
```

Comment:
i know i was the one who said to use the term "GAME" here, but that was just me thinking on the spot :) let's change this to VALID_MATCH please

Reply:
Also, this comment is a bit odd, can you say "used for filtering out games without 4 players"

Reply:
Done and done - also changed the comment to refer to matches instead of games.